platform/util/platform-util-unavailableurls/pom.xml

```diff
@@ -24,6 +26,11 @@
     <packaging>jar</packaging>
     <dependencies>
         <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
Do we want to pull in a new dependency for this? Seems easy enough to set the system property in the setup method and then revert it in a teardown method after the test.